apps/frontend/src/lib/auth/register.ts

```diff
@@ -207,6 +216,18 @@ export async function applyBackupsToNewUser(password: string): Promise<void> {
           case BackupEntryType.CONNECTION:
             const connection = ConnectionSchema.parse(JSON.parse(backup.backup));
 
+            // Give onboarding credit to first user connection
+            if (!gaveOnboardingCredit) {
+              await updateLeaderboardEntry({
+                authToken: session.authTokenValue,
+                chipIssuer: ChipIssuer.DEVCON_2024, // TODO: make this generic
```

Comment:
should make a note here to update for ETH India

Reply:
I improved this -- I pass in the chipIssuer corresponding to the tap.